Fedora/hpx.spec

```diff
@@ -201,11 +259,24 @@ for mpi in '' openmpi mpich ; do
   test -n "${mpi}" && module unload mpi/${mpi}-%{_arch}
 done
 
+%post -p /sbin/ldconfig
+
+%postun -p /sbin/ldconfig
+
+%post mpich -p /sbin/ldconfig
+
+%postun mpich -p /sbin/ldconfig
+
+%post openmpi -p /sbin/ldconfig
+
+%postun openmpi -p /sbin/ldconfig
```

Comment:
I think you want to use  `%ldconfig_scriptlets` instead of `%post` and `%postun`, also mpi packages need now `post` and `postun`, see https://src.fedoraproject.org/rpms/boost/blob/f27/f/boost.spec#_1139

Reply:
Ok, I will remove the mpi packages

Reply:
I added `%ldconfig_scriptlets libs` instead of post and postun